programs/temp/src/instructions/update_global_state.rs

```diff
@@ -0,0 +1,31 @@
+use crate::seeds::GLOBAL_STATE_SEED;
+use crate::state::{GlobalState, GlobalStateInput};
+use anchor_lang::prelude::*;
+
+#[derive(Accounts, Clone)]
+#[instruction(state: GlobalStateInput)]
+pub struct CreateGlobalState<'info> {
+    #[account(
+        mut
+        constraint = authority.key() == state.authority @ ErrorCode::InvalidUpdateAuthority,
+    )]
+    pub authority: Signer<'info>,
```

Comment:
Not in this PR, but could you please add a separate payer account to the create_global_state instruction so that we can separate the rent payer from the authority? Normally they are the same but this is good practice and matches similar instructions in the sunrise app.